build/nv.d3.js

```diff
@@ -12324,7 +12324,10 @@ nv.models.stackedArea = function() {
                 .width(availableWidth)
                 .height(availableHeight)
                 .x(getX)
-                .y(function(d) { return d.display.y + d.display.y0 })
+                .y(function(d) {
+                  if( typeof d.display === 'undefined' ) {}
+                  else { return d.display.y + d.display.y0 }
```

Comment:
Ah yes, that's much better, you might need the "typeof" in front of d.display but I'm not sure. The issue was https://github.com/novus/nvd3/issues/1093 not #1193, as you can see I was quite tired when I did this...